### ISSUE TITLE
Bug in Beta 3.0.0 with light groups and entity_id attribute

### ISSUE THREAD

**Issue by klatka** [jseidl/hass-magic_areas#206]
While using 3.0.0-beta2 I noticed a bug in attributes of created light groups.
Instead of covers, there should be an attribute called `entity_id` containing a list of lights.

Example: `light.lounge_lights`

```
min_mireds: 153
max_mireds: 500
supported_color_modes:
  - color_temp
  - xy
color_mode: color_temp
brightness: 128
color_temp: 397
hs_color:
  - 28.827
  - 71.874
rgb_color:
  - 255
  - 159
  - 71
xy_color:
  - 0.546
  - 0.389
covers:
  - cover.level_schlafzimmer
  - cover.level_wohnbereich
  - cover.level_buro
  - cover.level_lounge
icon: mdi:lightbulb-group
friendly_name: Licht in der Lounge
supported_features: 40
```

Strangely enough, it works correctly with the meta areas.

See `light.interior_lights`

```
supported_color_modes:
  - brightness
  - color_temp
  - rgbw
  - xy
icon: mdi:lightbulb-group
friendly_name: Licht drinnen
supported_features: 44
min_mireds: 125
max_mireds: 666
effect_list:
  - None
  - blue_fade
  - blue_strobe
  - candle
  - colorjump
  - colorloop
  - colorstrobe
  - cyan_fade
  - cyan_strobe
  - fire
  - gb_cross_fade
  - green_fade
  - green_strobe
  - purple_fade
  - purple_strobe
  - random
  - rb_cross_fade
  - red_fade
  - red_strobe
  - rg_cross_fade
  - white_fade
  - white_strobe
  - yellow_fade
  - yellow_strobe
color_mode: color_temp
brightness: 120
color_temp: 456
hs_color:
  - 29.826
  - 85.007
rgb_color:
  - 255
  - 145
  - 38
xy_color:
  - 0.581
  - 0.387
effect: None
entity_id:
  - light.balkenbeleuchtung
  - light.esstischdecke
  - light.bad_spot_1
  - light.hue_go_1
  - light.gaste_bad_spot_1
  - light.hue_play_1
  - light.gaste_bad_spot_2
  - light.flur_spot_2
  - light.hue_play_2
  - light.galerie
  - light.nachttisch_rechts
  - light.bad_spot_2
  - light.flur_spot_1
  - light.buro_spot_1
  - light.esstisch
  - light.arbeitsplatte
  - light.flur_spot_3
  - light.buro_spot_2
  - light.abstellmannchen
  - light.nachttisch_links
  - light.stehlampe_im_buro
```

In lights.py the light group for meta areas is created with
```
        light_groups.append(
            LightGroup(unique_id, f"{area.name} Lights", light_entities)
        )
```

while light groups for normal areas area created with class `AreaLightGroup` and line
`
        LightGroup.__init__(self, unique_id, self._name, self._entities)
`

**Comment by jseidl** [comment]
Hmmm I saw that bug before, maybe I just fixed it for meta areas 🤷 

Thanks for the report, will look into it

**Comment by jseidl** [comment]
That should do it https://github.com/jseidl/hass-magic_areas/pull/207

Probably (python, yay!) the reference is being overwritten by the cover module. I'm now correctly initializing the attributes to blank then adding the lights there.

**Comment by jseidl** [comment]
Fixed on 3.0. If the issue persists, please reopen!
